src/Playwright.CLI/Playwright.CLI.csproj

```diff
@@ -6,7 +6,7 @@
     <Summary>The Playwright CLI dotnet tool.</Summary>
     <Description>Playwright enables reliable end-to-end testing for modern web apps. It is built to enable cross-browser web automation that is ever-green, capable, reliable and fast. Learn more at https://playwright.dev/dotnet/.</Description>
     <PackageIcon>icon.png</PackageIcon>
-    <TargetFrameworks>net5.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
```

Comment:
not sure about this one, dotnet-format e.g. does just net6.0 there: https://github.com/dotnet/format/blob/3f3a6d862960df56a7fae32f41e9c019930080c5/src/dotnet-format.csproj#L4